deploy/stratos-base-images/Dockerfile.stratos-ui-base.tmpl

```diff
@@ -0,0 +1,8 @@
+FROM {{BASE_IMAGE}}
```

Comment:
Rename to Dockerfile.stratos-ui-build-base.tmpl

and update dependencies on this name.

Reply:
Updated!